site/docs/id/guide/basics.md

```diff
@@ -39,10 +39,13 @@ Semua method yang dapat digunakan oleh bot (**[daftar penting](https://core.tele
 ```ts
 // Mengirim sebuah pesan ke pengguna 12345.
 await bot.api.sendMessage(12345, "Halo!");
-// Selain itu, kamu bisa menambahkan opsi ke object.
+// Selain itu, kamu juga bisa menambahkan beberapa opsi dalam bentuk object.
 await bot.api.sendMessage(12345, "Halo!", {/* opsi lainnya */});
+// Memeriksa object message dari pesan yang telah terkrim.
```

Comment:
```suggestion
// Memeriksa object message dari pesan yang telah terkirim.
```

Reply:
Do we need `telah terkirim`? or just `terkirim`? It's past tense sentence, what about `dikirim`? How?

Reply:
Shouldn't it be in the past tense because the message is sent first and then print the response?

Reply:
I think all of them is right. Maybe we can use `terkirim` or `dikirim` to make it shorter.

Reply:
I updated that **suggestion** use `dikirim` word for preview. What do you think?

Reply:
Sounds good